bin/configure.sh

```diff
@@ -12,17 +12,12 @@ if [[ -z "$AWS_REGION" ]]; then
     AWS_REGION=ap-northeast-1
 fi
 
-cd lambda
-npm install
-zip -r lambdafunction.zip ./*
-aws s3 cp lambdafunction.zip s3://${S3_BUCKET_NAME}/
-rm lambdafunction.zip
-cp ../pipeline.yaml .
-sed -i -e "s@S3_BUCKET_NAME@${S3_BUCKET_NAME}@g" pipeline.yaml
-aws s3 cp pipeline.yaml s3://${S3_BUCKET_NAME}/
-rm pipeline.yaml
-rm pipeline.yaml-e
-cd ..
+cp pipeline.yaml bin/
+sed -i -e "s@ECS_REGION@${AWS_REGION}@g" bin/pipeline.yaml
+sed -i -e "s@S3_BUCKET_NAME@${S3_BUCKET_NAME}@g" bin/pipeline.yaml
+aws s3 cp bin/pipeline.yaml s3://${S3_BUCKET_NAME}/
+rm bin/pipeline.yaml
+rm bin/pipeline.yaml-e
```

Comment:
How about adding a `set -e` so that the bash exits if any of the commands fails. Right now, even if any error occurs, such as in S3 upload, URL to launch CF stack will be generated.